libs/core/src/git.ts

```diff
@@ -41,7 +41,7 @@ export function getMergeBase({
 
 export function getDiff({ base, cwd }: BaseGitActionArgs): string {
   try {
-    return execSync(`git diff ${base} --unified=0`, {
+    return execSync(`git diff ${base} --unified=0 --relative -- ${cwd}`, {
```

Comment:
shouldn't that be:
```suggestion
    return execSync(`git diff ${base} --unified=0 --relative="${cwd}"`, {
```
?

Reply:
OK tried running it locally and it seems like you're right :)